### PR TITLE
Add and delete notes functionalities

### PR DESCRIPTION
**Pull request by gadgetfather**
I have added add note and delete note functionality please review it.
Here is preview link: https://deploy-preview-3--chic-daifuku-c6425e.netlify.app/notes
add and delete note

### CODE REVIEW

**Review by Veekshith-Thoram — APPROVED** [review]
The pr is fine.

File: src/context/note-context.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  const addNote = async (noteText) => {
+    try {
+      const response = await axios.post(
+        "/api/notes",
+        { note: noteText },
+        { headers: { authorization: token } }
+      );
+
+      const {
+        data: { notes },
```
Comment (mohammed-sas) [inline src/context/note-context.js:20]: Check for response status before updating state

**Review by mohammed-sas — APPROVED** [review]
Functionalities are working as intended
Note context has been coded good
Files are well organised
Other than suggested your pr is good to merge

File: src/context/note-context.js (hunk 1 above)
Comment (gadgetfather) [reply src/context/note-context.js:20]: Thanks, i will do it 👍

**Pull request merged**